SimCalorimetry/EcalSimProducers/python/esCATIAGainProducer_cfi.py

```diff
@@ -0,0 +1,9 @@
+import FWCore.ParameterSet.Config as cms
+
+EcalCATIAGainRatiosRcd =  cms.ESSource("EmptyESSource",
+                                recordName = cms.string("EcalCATIAGainRatiosRcd"),
+                                firstValid = cms.vuint32(1),
+                                iovIsRunNotTime = cms.bool(True)
+                                )
+
+EcalCATIAGainRatios = cms.ESProducer("EcalCATIAGainRatiosESProducer"),
```

Comment:
remove trailing comma

Reply:
not done

Reply:
because if i remove the trailing comma i get the error:
----- Begin Fatal Exception 21-Jul-2020 16:40:58 CEST-----------------------
An exception of category 'Configuration' occurred while
   [0] Constructing the EventProcessor
   [1] Constructing ESProducer: class=EcalCATIAGainRatiosESProducer label='EcalCATIAGainRatios'
Exception Message:
MissingParameter: Parameter 'ComponentName' not found.
----- End Fatal Exception -------------------------------------------------